### PR TITLE
Update ingesting-data.md

### PR DESCRIPTION
**Pull request by mcallica**
Edits include adding a section on using the PostgreSQL and TimescaleDB output plugin for Telegraf and using PostgreSQL's Kafka connector.

### CODE REVIEW

File: using-timescaledb/ingesting-data.md [hunk 1]
```diff
@@ -28,10 +28,52 @@ TimescaleDB. Read more about how to set up Prometheus with TimescaleDB in our
...
+modification. This means that as metrics are collected by Telegraf, the plugin creates a table if it doesn’t exist and alters
+the table if a schema has changed. By default, the plugin leverages a [wide model][wide-model], which is typically the schema
+model that TimescaleDB users tend to choose when storing metrics. However, you can specify that you want to store metrics in a
+narrow model with a separate metadata table and foreign keys. You can also choose to use JSONB.
+
+To get started with the PostgreSQL and TimescaleDB output plugin, visit the [tutorial][telegraf-tutorial]. 
+
+## PostgreSQL's Kafka connector [](postgresqls-kafka-connector)
+
+Another popular method of ingesting data intoTimescaleDB is through the use of the [PostgreSQL connector with Kafka Connect][postgresql-connector-with-kafka-connect]. The connector is designed to work with [Kafka Connect][kafka-connect] and to be deployed to a Kafka Connect runtime service. It’s purpose is to ingest change events from PostgreSQL databases (i.e. TimescaleDB).
```
Comment (svenklemm) [inline using-timescaledb/ingesting-data.md]: missing space intoTimescaleDB

Comment (mcallica) [reply using-timescaledb/ingesting-data.md]: done!

File: using-timescaledb/ingesting-data.md [hunk 2]
```diff
@@ -28,10 +28,52 @@ TimescaleDB. Read more about how to set up Prometheus with TimescaleDB in our
...
+
+We wrote the PostgreSQL output plugin which also has the ability to send data to a TimescaleDB hypertable. Telegraf handles 
+batching, processing, and aggregating the data collected prior to inserting that data into TimescaleDB.
+
+<!-- -->
+>:WARNING: The [pull request][pull-request] is open and currently under review by the Telegraf developers, waiting to be 
+merged. To give users the opportunity to try this functionality, we built [downloadable binaries][downloadable-binaries] of
+Telegraf with our plugin already included. 
+
+The PostgreSQL plugin extends the ease of use users get from leveraging Telegraf by handling schema generation and
```
Comment (svenklemm) [inline using-timescaledb/ingesting-data.md]: hmm this is not entirely correct. not sure we wanna rewrite though. a "pure" narrow model usually has one value per row but with the plugin you will have multiple values per row unless you set fields as jsonb, the tag configuration is more about storing tags normalized/denormalized.

Comment (dianasaur323) [reply using-timescaledb/ingesting-data.md]: We use this verbage elsewhere. A fix could be to say: However, you can specify that you want to store metrics and their tags in a normalized manner, with a separate metadata table and foreign keys. 

I think this is a little confusing though, no?

**Comment by solugebefola** [comment]
Can you add a version label?  Thx

**Pull request merged**